### PR TITLE
Clarified salaried routes

### PR DESCRIPTION
**Pull request by tcannon3**
Clarified some language around salaried routes. Highlighted there is a difference between school direct (salaried) and school direct. Highlighted that places are limited and competitive

### Trello card
https://trello.com/c/TG0Ln7dX/2656-review-content-around-eligibility-and-opportunities-available-for-salaried-courses

### CODE REVIEW

File: app/views/content/ways-to-train.md [hunk 1]
```diff
@@ -148,13 +148,17 @@ Fees are usually around £9,250 but you can:
...
 
-These courses are usually suitable for people who already have relevant teaching experience, or are already working as unqualified teachers. 
+These salaried routes are usually suitable for people with a degree and relevant school experience. 
 
 Many 'school direct (salaried)' programmes give you a PGCE or PGDE (an academic qualification) in addition to QTS.
 
-Courses typically take one year but you may be able to study part-time.
+It can be easy to confuse the 'school direct (salaried)' programme with the non-salaried 'school direct' programme. Make sure you check with your chosen provider before applying.
+
+Courses usually take one year but you may be able to study part-time.
```
Comment (peteryates) [inline app/views/content/ways-to-train.md:159]: This sentence feels a little odd to me, the second part doesn't feel related to the first.

Perhaps:

```suggestion
Courses usually take one year but you may take two if you study part-time.
```
Just an example, don't know if it's even true! 😅

Comment (gemmadallmandfe) [reply app/views/content/ways-to-train.md:159]: Not sure I agree with your suggestion @peteryates but will leave for @Tomcowan1 as I know that he, Ellie and Tara are looking at broader changes to the Ways to train page as a result of TTA feedback

**Comment by tcannon3** [comment]
> Hello - apols but I can't find this one on the Trello B - is this something that's been requested recently? A bit of background would be V helpful - thanks :-)

Hi, it's here on Trello. It's regarding feedback that Gaye received at the Mcr Train to Teach event https://trello.com/c/TG0Ln7dX/2656-review-content-around-eligibility-and-opportunities-available-for-salaried-courses

File: app/views/content/ways-to-train.md (hunk 1 above)
Comment (tcannon3) [reply app/views/content/ways-to-train.md:159]: Maybe: "Full-time courses usually take one year to complete but you may be able to study part-time." 
Or: "Courses are usually full-time and take one year to complete. You may be able to study part-time."

**Comment by gemmadallmandfe** [comment]
Thanks for following up on the comments @tcannon3 

Having discussed with @Tomcowan1, this PR has now been superseded by https://github.com/DFE-Digital/get-into-teaching-app/pull/2163 so I'm going to delete it.

**Pull request closed without merging**